### ISSUE TITLE
(v3.11.2.1) Xowa still tries to download -latest- files even though the -latest- (or -date-) files are already inside the import folders.

### ISSUE THREAD

**Issue by khanhhung2512** [gnosygnu/xowa#101]
I just tested the newest release and the issue still persists.

**Comment by gnosygnu** [comment]
Thanks. You're right. I stared at the ticket for a moment, before I realized my misunderstanding

I changed tonight's release to not download the files when rebuilding the category system. The following statement used to download the category file.

```
// rebuild category
app.setup.cmds.dump_add_many( 'en.wikinews.org', 'latest', '', 'wiki.category2.build');
```

However, I now understand that you're talking about the main import. Specifically, this line:

```
// import wiki from dump file
app.setup.cmds.cmd_add("wiki.dump_file", "D:\xowa\wiki\en.wikinews.org\enwikinews-latest-pages-articles.xml.bz2", "en.wikinews.org", "");
```

This does try to download the category files, even if you already have them.

I'll fix this in the next release. Let me know if you want it sooner, and I'll upload a preview release for you tomorrow / mid-week.

Also, FWIW, if you have any questions, I jumped on IRC now: http://webchat.freenode.net/?channels=#xowa

Thanks.
